### DISCUSSION TITLE
Custom asset support for stellar account viewer

### DISCUSSION THREAD

**Discussion by Kanaye** [kalepail/stellar-quest-bounties#81] [Bounty Ideas]
Currently Account viewer only supports native/XLM as the only asset.

I'd like the idea of at least supporting the basic operations for custom assets on account viewer, namely:
- Add a custom asset  trustline
- make a simple payment with the custom asset
- remove the trustline 
- show the current balance of custom assets and their history

Might be out of scope for bounties or for account viewer, so 'we need to think about this and maybe talk to the account viewer maintainers if we decide that this is a good bounty. 

Links:
- Account Viewer: https://accountviewer.stellar.org/
- Account Viewers Repo: https://github.com/stellar/account-viewer-v2

**Comment by kalepail** [comment]
Great idea! I'll direct that team to this discussion and let's see what they say.

**Reply by kalepail** [reply]
They love the idea. You'd just need to fork and implement the change vs requiring a PR. There's a chance it wouldn't be implemented directly as the team is pretty strict on the UX. You could go your own route though and then attempt a PR right into the repo separate from the bounty. Worst case though you have a great UI for personal use or maybe for other brave souls to run themselves. Get enough support and the AV team could use this as a base for their own implementation.

So if you're down to write the bounty I say go for it!
